src/sbt-test/sbt-compatibility/library/test

```diff
@@ -29,7 +41,6 @@
 # two patch versions
 -> v1-0-2/mimaReportBinaryIssues
 -> v1-0-2/compatibilityCheck
```

Comment:
This test is failing, it should pass once we check that source incompatibilities can’t be introduced between patch versions.